main.cpp

```diff
@@ -2276,19 +2283,25 @@ int main(int argc, char **argv) {
 #if defined(_WIN32)
                             printer(dr_vidpid_stdio_usb,
                                     " appears to be a RP2040 device with a USB serial connection, not in BOOTSEL mode. You can force reboot into BOOTSEL mode via 'picotool reboot -f -u' first.");
+                            printer(dr_reset_interface,
+                                    " appears to be a RP2040 device with a USB reset interface, not in BOOTSEL mode. You can force reboot into BOOTSEL mode via 'picotool reboot -f -u' first.");
```

Comment:
Should the example picotool command-line here also include the `-i` flag?

Reply:
Right. The users will only see this message, if they already are using '-i' option. Do we want to promote the possibility of using the option similar to how we promote use of `-f` option?